collect_app/build.gradle

```diff
@@ -361,8 +361,12 @@ dependencies {
     testImplementation "org.mockito:mockito-core:3.7.7"
     testImplementation 'org.mockito:mockito-inline:3.7.7'
 
-    testImplementation project(path: ':testshared')
-    testImplementation "org.robolectric:robolectric:4.3.1"
+    testImplementation(project(path: ':testshared')) {
+        exclude group: 'org.robolectric' // Some tests in `collect_app` don't work with newer Robolectric
```

Comment:
This problem is best left to our next "update all the libraries" cycle as it will involve reworking a few unrelated tests.